docker-compose.yml

```diff
@@ -144,8 +150,11 @@ services:
     restart: on-failure
 
   browserless:
+    image: ecamp/ecamp3-browserless
```

Comment:
Du buildest und pushst nirgends ein Image mit diesem Namen. Entweder auch noch in der GitHub Action builden oder diese Modifikation wieder entfernen.

Reply:
Done